tools/run_tests/xds_k8s_test_driver/kubernetes-manifests/client-secure.deployment.yaml

```
@@ -63,8 +63,8 @@ spec:
             % if xds_server_uri:
             - "--xds-server-uri=${xds_server_uri}"
             % endif
-            % if config_scope:
-            - "--config-scope-experimental=${config_scope_experimental}"
+            % if config_mesh:
+            - "--config-mesh-experimental=${config_mesh_experimental}"
```

```suggestion
            - "--config-mesh-experimental=${config_mesh}"
```

I think this was wrong in the first place, there isn't a variable `config_scope_experimental` set `framework.test_app.client_app.KubernetesClientRunner.run`. But we probably never ran security tests with the app net on.

---

Ooh. Good catch. Fixed.